src/mono/wasm/debugger/DebuggerTestSuite/EvaluateOnCallFrameTests2.cs

```diff
@@ -197,10 +197,18 @@ public static IEnumerable<object[]> InstanceMethodForTypeMembersTestData(string
             wait_for_event_fn: async (pause_location) =>
             {
                 var id = pause_location["callFrames"][0]["callFrameId"].Value<string>();
+                var id_prev = pause_location["callFrames"][1]["callFrameId"].Value<string>();
 
                 var (_, res) = await EvaluateOnCallFrame(id, "DebuggerTests.EvaluateStaticFieldsInStaticClass.StaticProperty2", expect_ok: false);
                 AssertEqual("Failed to resolve member access for DebuggerTests.EvaluateStaticFieldsInStaticClass.StaticProperty2", res.Error["result"]?["description"]?.Value<string>(), "wrong error message");
-
+                Assert.Equal("DebuggerTests.EvaluateMethodTestsClass.TestEvaluate.run", res.Error["exceptionDetails"]?["stackTrace"]?["callFrames"]?[0]?["functionName"]?.Value<string>());
```

Comment:
Extract `res.Error["exceptionDetails"]?["stackTrace"]?["callFrames"]?[0]` to a local. it would help the readability also.